code/modules/power/generators/turbine.dm

```diff
@@ -26,8 +26,11 @@
 #define FAST 2
 #define SLOW 1
 
+//below defines the time between an overheat event and next startup
+#define OVERHEAT_TIME 120
```

Comment:
```suggestion
#define OVERHEAT_TIME 12 SECONDS
```

Reply:
For whatever reason, this value is already in seconds, as the turbine processes once a second (this is 120 seconds, or two minutes). This value is used by the var/last_overheat to countdown until the turbine can be reenabled. would you be opposed to noting the value in seconds as a comment above or to the side of the define?

Reply:
yeah that works